packages/http/src/resources/__tests__/Hotspots.spec.ts

```diff
@@ -283,6 +283,23 @@ describe('listJson', () => {
   })
 })
 
+describe('search by hotspot name', () => {
+  nock('https://api.helium.io')
+    .get('/v1/hotspots/name')
+    .query({ search: 'chicken-burrito' })
+    .reply(200, {
+      data: [hotspotFixture({ name: 'chicken-burrito-guacamole' }), hotspotFixture({ name: 'chicken-burrito-salsa' })],
+    })
+
+  it('lists hotspots', async () => {
+    const client = new Client()
+    const list = await client.hotspots.list({ search: 'chicken-burrito' })
```

Comment:
i'm just wondering if we want to overload the `list` function here, or maybe create a separate `search` function so the syntax would be like

```ts
client.hotspots.search({ name: 'chicken-burrito' })
```

I'm also ok with it as it is, just maybe something to consider

Reply:
I like that idea. If that feels better to you, I will update it.

Reply:
yeah I guess it might also help simplify the growing list function, if we separated out search. it seems a little complex right now, and maybe odd to need to separate out the cursor from the other params

Reply:
cool. I'll change it.